new-dti-website/components/initiative/InitiativeDisplay.tsx

```diff
@@ -0,0 +1,102 @@
+import Image, { ImageProps } from 'next/image';
+import { Inter } from 'next/font/google';
+import initiatives from './data/initiatives.json';
+import useScreenSize from '../../src/hooks/useScreenSize';
+import { LAPTOP_BREAKPOINT } from '../../src/consts';
+import { ibm_plex_mono } from '../../src/app/layout';
+
+const inter = Inter({ subsets: ['latin'] });
+
+type InitiativeProps = {
+  title: string;
+  subtitle: string;
+  icon: ImageProps;
+  image: ImageProps;
+  description: string;
+  eventName: string;
+  className?: string;
+};
+
+const Initiative = ({
+  title,
+  subtitle,
+  icon,
+  image,
+  description,
+  eventName,
+  className
+}: InitiativeProps) => {
+  const parts = description.split(new RegExp(`(${eventName})`));
+  const { width } = useScreenSize();
+
+  return (
+    <div className={`flex flex-col gap-10 ${className}`}>
+      <div className="flex flex-col gap-[14px]">
+        <div className="flex gap-2 min-h-10 xs:flex-col lg:flex-row">
+          <Image {...icon} />
+          <h3 className="font-semibold text-[32px] leading-10">{title}</h3>
+        </div>
+        <p className="font-semibold text-[22px] leading-[26px]">{subtitle}</p>
+        <div className="flex justify-center overflow-hidden rounded-md">
+          <Image {...image} className="h-[260px] max-w-none" />
```

Comment:
From the Figma, the images should be a bit more rounded and have a drop shadow! I see that you used rounded-md but it doesn't show in the video you provided! Looking Great though!

Reply:
Also, I don't know if this was intended but some of the images also appear to have a border in the figma. Might be good to ask Vannessa about this.

Reply:
+1, I think the intention was to make these rounded